src/components/pages/userDetailPage.tsx

```diff
@@ -1,7 +1,44 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+
+import { User } from 'devu-shared-modules'
 
 import PageWrapper from 'components/shared/layouts/pageWrapper'
+import LoadingOverlay from 'components/shared/loaders/loadingOverlay'
+
+import { useParams } from 'react-router-dom'
+
+import RequestService from 'services/request.service'
+
+import EditUserForm from 'components/shared/forms/editUserForm'
+
+import styles from './userDetailPage.scss'
+
+type Params = {
+  userId: string
+}
+
+const UserDetailPage = ({}) => {
+  const { userId } = useParams() as Params
+
+  const [loading, setLoading] = useState(true)
+  const [user, setUser] = useState({} as User)
+
+  useEffect(() => {
+    RequestService.get(`/api/users/${userId}`)
+      .then(setUser)
+      .catch((e) => console.error(e))
+      .finally(() => setLoading(false))
+  }, [])
+
+  if (loading) return <LoadingOverlay />
 
-const UserDetailPage = ({}) => <PageWrapper>User Detail Update</PageWrapper>
+  return (
+    <PageWrapper>
+      <div className={styles.userDetailDiv}>
```

Comment:
Idk how to feel about this class name.

Reply:
updated it to something i think is more descriptive but its still just a how you feel about class names sort of thing